azkaban-db/src/main/java/azkaban/db/AzDBUtil.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2017 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ */
+package azkaban.db;
+
+public class AzDBUtil {
```

Comment:
call it `Constants`? I specifically avoid creating `Util` classes which become a static dumpground for people to dump random unrelated methods into.

I think we are redoing some of the infrastructure in this module. How about @HappyRay 's idea of pulling out a core package and depending on that?